platooning_strategic_IHP/include/platoon_strategic_ihp.h

```diff
@@ -149,6 +150,21 @@ namespace platoon_strategic_ihp
             */
             cav_msgs::Maneuver composeManeuverMessage(double current_dist, double end_dist, double current_speed, double target_speed, int lane_id, ros::Time& current_time);
 
+            /**
+            * \brief Find start(current) and target(end) lanelet index from path to generate lane change maneuver message.
+            * 
+            * \param current_dist current downtrack distance
```

Comment:
Please add units of measure to all these param descriptions.  Speed, especially, can be uncertain since Americans tend to talk in mph.